components/brave_ads/browser/notification_helper.h

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BROWSER_BRAVE_ADS_NOTIFICATION_HELPER_H_
+#define BRAVE_COMPONENTS_BROWSER_BRAVE_ADS_NOTIFICATION_HELPER_H_
+
+#include <string>
+
+#include "base/macros.h"
+#include "base/memory/singleton.h"
+#include "build/build_config.h"
+
+namespace brave_ads {
+
+class NotificationHelper {
+ public:
+  
+  static NotificationHelper* GetInstance();
+  
+  virtual bool IsNotificationsAvailable() const;
+
+  virtual bool IsBraveAdsChannelEnabled() const;
+
+  virtual void OpenPageFromNative(const std::string & url) const;
```

Comment:
This is platform specific so should not be a part of this class

Reply:
I think, `OpenPageFromNative` should be required by all platforms. I will check if we have it already implemented in base code.